sim/midas/src/main/scala/midas/stage/Annotations.scala

```diff
@@ -45,3 +48,16 @@ object RuntimeConfigNameAnnotation extends HasShellOptions {
       shortOption = Some("ggrc"),
       helpValueName = Some("<filename>") ) )
 }
+
+case class OutputBaseFileNameAnnotation(name: String) extends NoTargetAnnotation with GoldenGateOption
+
+object OutputBaseFileNameAnnotation extends HasShellOptions {
+  val options = Seq(
+    new ShellOption[String](
+      longOption = "--base-output-filename",
+      toAnnotationSeq = (a: String) => Seq(OutputBaseFileNameAnnotation(a)),
+      helpText = "Specifies the base (prefix) used on most Golden Gate generated files.",
```

Comment:
Again, why 'most' and not just "all"?  Does it really add something other than potential confusion and having to remember exceptions to the rule to have the runtime.conf exception?

Reply:
runtime.conf. I'll fix it though.